src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -330,6 +330,7 @@ protected virtual bool IsAttributeConditionallyOmitted(NamedTypeSymbol attribute
                         }
 
                         BoundExpression boundNamedArgument = BindNamedAttributeArgument(argument, attributeType, diagnostics);
+                        this.Compilation.VariableUsePass.Visit(boundNamedArgument);
```

Comment:
Attributes in local functions result in `error CS8205: Attributes are not allowed on local function parameters or type parameters`, so eliminating the warning isn't needed.